src/main/java/com/example/gsheets/controller/dto/StudentDTOController.java

```diff
@@ -1,12 +1,12 @@
-package com.example.gsheets.controller;
+package com.example.gsheets.controller.dto;
 
 import lombok.Builder;
 import lombok.Data;
 
 @Data
 @Builder
-public class StudentDTO {
-    private String Id;
+public class StudentDTOController {
```

Comment:
DTO should be in the end

Reply:
should be without "Controller"

Reply:
I used this variant namespace because i have 2 DTO (Controller and Client) and if i use "StudentDTO" i don't know what exactly DTO i mean

Reply:
but i think better will be conversation with this question

Reply:
done